css-two-column-layout/styles.css

```diff
@@ -74,3 +72,8 @@ button {
   padding: 6px 24px;
   margin-top: -2px;
 }
+
+.row-column {
+  display: flex;
+  justify-content: center;
+}
```

Comment:
You should reuse the `row` class instead of making variations on it. You can make an additional class with `justify-content: center` to add to the element in addition to the `row` class.